recipe/meta.yaml

```diff
@@ -1,49 +1,48 @@
-{% set version = "4.65.0" %}
+{% set version = "4.66.2" %}
 
 package:
   name: tqdm
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/t/tqdm/tqdm-{{ version }}.tar.gz
-  sha256: 1871fb68a86b8fb3b59ca4cdd3dcccbc7e6d613eeed31f4c332531977b89beb5
+  sha256: 6cd52cdf0fef0e0f543299cfc96fec90d7b8a7e88745f411ec33eb44d5ed3531
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - tqdm = tqdm.cli:main
+  skip: true  # [py<37]
 
 requirements:
   host:
     - python
     - pip
-    - setuptools_scm
+    - setuptools_scm >=3.4
     - setuptools
     - toml
     - wheel
   run:
     - python
     - colorama  # [win]
   run_constrained:
-    - slack-sdk
-    - requests
     - ipywidgets >=6
 
 test:
   requires:
-    - dask
-    - keras
```

Comment:
One of the tests still requires `keras` but it's skipped so it's up to you if  we need it in the test environment:
`SKIPPED [1] tests/tests_keras.py:9: could not import 'tqdm.keras': No module named 'keras'`

Reply:
I left it out on purpose as old versions of keras would be picked-up otherwise (because of the dependency on tensorflow).